src/components/search/AnalysesSearchResults.js

```diff
@@ -0,0 +1,128 @@
+/**
+ *
+ * Display tablular view of detailed analyses search results
```

Comment:
I saw you fixed this typo in another file, but it's still here at least, possibly in other files as well

Reply:
Fixed in #170 :)